src/components/ReactSelect/_react-select.scss

```diff
@@ -51,6 +51,7 @@
 
     .#{$prefix}--react-select__option--is-focused {
       background: var(--ui-02);
+      color: var(--text-01)
```

Comment:
It could sometimes break the compiler if the `;` is missing.

Reply:
thanks for point out. WIll resolve that